scenarios/sample-download.xml

```diff
@@ -32,8 +32,8 @@
                 <class>com.sonatype.nexus.perftest.maven.DownloadOperation</class>
                 <repo>public</repo>
                 <paths>
-                  <class>com.sonatype.nexus.perftest.maven.HttpdLogParser</class>
-                  <logfile>data/maven-3.1-build-artifact-access.log.gz</logfile>
+                  <class>com.sonatype.nexus.perftest.maven.CsvLogParser</class>
+                  <logfile>data/maven-3.3.4-build-artifact-access.csv.gz</logfile>
```

Comment:
Ah... the latest official Maven release on central is 3.3.3 - the 3.3.4 release should be out in the next few weeks, but it has yet to be staged and put up to vote: http://www.mail-archive.com/dev@maven.apache.org/msg106201.html